src/main/java/step5/util/Game.java

```diff
@@ -0,0 +1,78 @@
+package step5.util;
+
+import step5.domain.RacingCar;
+import step5.domain.UserInput;
+import step5.view.InputView;
+import step5.view.ResultView;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Game {
+    public static void start() {
+        //inputView
+        final UserInput userInput = InputView.input();
+
+        List<RacingCar> cars = makeCars(userInput.getCarNames());
+
+        //resultView
+        ResultView.printResultText();
+
+        for (int i = 0; i < userInput.getTryNumber(); i++) {
+            tryMove(cars);
+            ResultView.printTryResult(cars);
+        }
+
+        ResultView.printWinners(toNames(extractWinners(cars)));
+    }
+
+    public static List<RacingCar> makeCars(final String[] carNames) {
```

Comment:
```suggestion
    private static List<RacingCar> makeCars(final String[] carNames) {
```

private 이 더 적절해보입니다.

테스트를 위해서라면 makeCars를 호출하는 start를 테스트하는 간접적인 방식으로 진행해주세요.